support-frontend/conf/DEV.public.conf

```diff
@@ -13,8 +13,8 @@ identity.useStub=false
 guardianDomain=".thegulocal.com"
 
 # To run selenium post-deployment tests against code use the config below
-# support.url="https://support.code.dev-theguardian.com"
-support.url="https://support.thegulocal.com"
+support.url="https://support.code.dev-theguardian.com"
+# support.url="https://support.thegulocal.com"
```

Comment:
I guess we don't want to commit this change?

Reply:
Whoops! Fixed.